### PR TITLE
Add the puppetlint trailing comma check

### PR DESCRIPTION
**Pull request by deanwilson**
In PR review missing trailing commas are raised and so rather
than having a person look for them we should let the computer do it.

This PR also bulk fixes the current missing commas.

### CODE REVIEW

**Comment by benlovell** [comment]
Did you mean puppet lint, not pupptline?

**Comment by alexmuller** [comment]
Looks good to me. Does anyone object to this being added to the build?

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "rake"
 gem "puppet-syntax", '2.1.0'
 gem "puppet-lint"
+gem 'puppet-lint-trailing_comma-check', :require => false
```
Comment (mattbostock) [inline Gemfile]: Nice use of `require`, hadn't seen that before :+1:

**Comment by mattbostock** [comment]
:+1: This is great. It would be good to say in the commit message why we like to use trailing commas.

**Comment by alexmuller** [comment]
I've updated this so if tests pass it should be merged :+1:

**Comment by mattbostock** [comment]
:tada:

**Pull request merged**